draft-ietf-httpbis-message-signatures.md

```diff
@@ -377,8 +377,8 @@ The following sections describe each of these steps in detail.
 5. The signer creates an ordered list of content identifiers representing the message content and signature metadata to be covered by the signature, and assigns this list as the signature's Covered Content.
    * Each identifier MUST be one of those defined in Section 2.
    * This list MUST NOT be empty, as this would result in creating a signature over the empty string.
-   * If the signature's Algorithm name does not start with rsa, hmac, or ecdsa, signers SHOULD include `*created` and `*request-target` in the list.
-   * If the signature's Algorithm starts with rsa, hmac, or ecdsa, signers SHOULD include `date` and `*request-target` in the list.
+   * Signers SHOULD include `*created` and `*request-target` in the list.
+   * Signers SHOULD include `date` and `*request-target` in the list.
```

Comment:
```suggestion
```

1. Signers may not have control over the `Date` header. (pre-signed messages, XmlHttpRequest callers, etc.)
2. The `*request-target` guidance is redundant with the line above.